Update README.md (#2)
rp-magrathea/yuggoth: README.md

#Yuggoth
A collection of blank maps, sorted by genre then type.

##Purpose

> The purpose of this repo is to provide maps for roleplay settings.

Reviewer: We could probably condense these first two sections into one. (It's pretty standard that the first line under the project name is the project's purpose from what I've seen.)

```suggestion
# Yuggoth
A collection of blank maps for roleplay settings, sorted by genre, then type.
```

> ##Commission Agreement

Reviewer: If I remember right, GitHub needs a space between the pound signs and the next word in order to display headings correctly.

```suggestion
## Commission Agreement
```

> For all commissioned maps, two copies will be made: the requester's version, including names, and our version, without names. Like our other resources, at the time tha t a site comes to a close, any maps that were made will be added to our collection and made available under a Creative Commons Zero license.

Reviewer: Caught an extra space trying to sneak in!

```suggestion
For all commissioned maps, two copies will be made: the requester's version, including names, and our version, without names. Like our other resources, at the time that a site comes to a close, any maps that were made will be added to our collection and made available under a Creative Commons Zero license.
```

> ##Disclaimers

Reviewer: ```suggestion
## Disclaimers
```

While all maps found within this repo are under the CC0 license, please keep in mind the following:

> A Creative Commons Zero license does not waive liability for use of copyrighted material. While these maps are open and available for use, modification, and distribution, both Inkarnate and Campaign Cartographer 3 retain copyrights to their artwork, icons, etc.  These individual images may not be distributed unless part of a larger work, i.e. a map.

Reviewer: This bit is worded really well, love it. <3
